services/thermal_mgr/thermal_mgr.c

```diff
@@ -41,20 +41,49 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 }
 
+#define ERR_CODE_FAIL 1
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
-  return ERR_CODE_SUCCESS;
+  if(xQueueSend(thermalMgrQueueHandle, event, portMAX_DELAY) == pdTRUE){
```

Comment:
Check that the event and queue handles aren't null, returning invalid arg and invalid state respectively